src/sidebar/util/session-util.js

```diff
@@ -10,6 +13,35 @@ function shouldShowSidebarTutorial(sessionState) {
   return false;
 }
 
+/**
+ * The following things must all be true for the tutorial component to auto-display
+ * on app launch:
+ * - The app must be operating within the "sidebar" (i.e. not single-annotation
+ *   or stream mode); AND
+ * - No configuration is present in `settings.services` indicating
+ *   that the host wants to handle its own help requests (i.e. no event handler
+ *   is provided to intercept the default help panel), AND
+ * - A user profile is loaded in the current state that indicates a `true` value
+ *   for the `show_sidebar_tutorial` preference (i.e. the tutorial has not been
+ *   dismissed by this user yet). This implies the presence of a profile, which
+ *   in turn implies that there is an authn'ed user.
+ *
+ * @param {boolean} isSidebar - is the app currently displayed in a sidebar?
+ * @param {Object} sessionState - `session` state from the store
+ * @param {Object} settings - app configuration/settings
+ * @return {boolean} - Tutorial panel should be displayed automatically
+ */
+function shouldAutoDisplayTutorial(isSidebar, sessionState, settings) {
+  const shouldShowBasedOnProfile =
+    typeof sessionState.preferences === 'object' &&
```

Comment:
The initial state of the `session` doesn't define a `preferences` object (doesn't get populated until profile loads). This strikes me as a little odd, but that's something for another day, perhaps.